packages/pulpcore/python-dotenv/python-dotenv.spec

```diff
@@ -31,8 +31,8 @@ Summary:        %{summary}
 rm -rf %{pypi_name}.egg-info
 
 # setup.py loads the contents of README.md and CHANGELOG.md into long_description
-# these files contain unicode charactes and thus fail to load in a non-unicode
-# environment like some EL7 builds
+# these files contain unicode charactes and thus fail to load in a non-unicode	
```

Comment:
This can be removed, it's only needed on Python <3.5

Reply:
eeeh, why is that on that line, it should be on the buildrequire line. damn GH android app.